specification/cognitiveservices/data-plane/AnomalyDetector/preview/v1.2-preview/RootCauseAnalysis.json

```diff
@@ -0,0 +1,2352 @@
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "query",
+            "name": "skip",
+            "description": "For paging, skipped number",
+            "type": "integer",
+            "format": "int32"
+          },
+          {
+            "in": "query",
+            "name": "maxpagesize",
+            "description": "The maximum number of items in one page",
+            "type": "integer",
+            "format": "int32"
+          },
+          {
+            "in": "query",
+            "name": "filter",
+            "description": "The filter expression filters out the resources to be returned, currently contains(displayName, 'filter name’) is supported",
+            "type": "string"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/RootCauseAnalysisInstanceResponseList"
+            }
+          },
+          "default": {
+            "description": "Client error or server error (4xx or 5xx)",
+            "schema": {
+              "$ref": "#/definitions/ErrorDetail"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },
+        "x-ms-examples": {
+          "Get instance list for root cause analysis": {
+            "$ref": "./examples/ListRootCauseAnalysisInstances.json"
+          }
+        }
+      }
+    },
+    "/rootCauseAnalysis/instances/{instanceId}/models/{modelId}": {
+      "put": {
+        "tags": [
+          "RootCauseAnalysisModel"
+        ],
+        "summary": "Trigger a model training task for root cause analysis",
+        "operationId": "CreateRootCauseAnalysisTrainingTask",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "path",
+            "name": "instanceId",
+            "description": "Instance unique id, should be a string of uuid",
+            "required": true,
+            "type": "string",
+            "format": "uuid"
+          },
+          {
+            "in": "path",
+            "name": "modelId",
+            "description": "Model unique id, should be a string of uuid",
+            "required": true,
+            "type": "string",
+            "format": "uuid"
+          },
+          {
+            "in": "body",
+            "name": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/RootCauseAnalysisTrainingRequest"
+            }
+          }
+        ],
+        "responses": {
+          "201": {
+            "description": "Success",
+            "headers": {
+              "Location": {
+                "description": "Location of the newly created resource.",
+                "type": "string"
+              }
+            }
+          },
```

Comment:
I am not sure whether I am seeing the correct spec file. This only return a "Location" header without any meaningful response body?

Check RELO pattern here https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md#long-running-operations--jobs

Reply:
added response with status as a field